### PR TITLE
[EHN] Including SCC

### PR DESCRIPTION
**Pull request by bruAristimunha**

### CODE REVIEW

File: braindecode/models/sccnet.py [hunk 1]
```diff
@@ -0,0 +1,176 @@
+"""Spatial component-wise convolutional network (SCCNet) for motor-imagery EEG
+    classification.
+Authors: Chun-Shu Wei
+         Bruno Aristimunha <b.aristimunha@gmail.com> (braindecode adaptation)
+
+Wei, C. S., Koike-Akino, T., & Wang, Y. (2019, March). Spatial component-wise
+convolutional network (SCCNet) for motor-imagery EEG classification. In 2019
+9th International IEEE/EMBS Conference on Neural Engineering (NER) (pp. 328-331).
+IEEE.
+"""
```
Comment (sylvchev) [inline braindecode/models/sccnet.py:10]:
```suggestion
# Authors: Chun-Shu Wei
#         Bruno Aristimunha <b.aristimunha@gmail.com> (braindecode adaptation)
#
# License: BSD (3-clause)
```

File: braindecode/models/sccnet.py [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+        components and temporal domain to capture spatio-temporal patterns.
+    3. **Temporal Smoothing (Pooling)**: Uses average pooling over time to smooth the
+       features and reduce the temporal dimension, focusing on longer-term patterns.
+    4. **Classification**: Flattens the features and applies a fully connected
+       layer.
+
+
+    Parameters
+    ----------
+    n_filters_spat : int, optional
```
Comment (sylvchev) [inline braindecode/models/sccnet.py:45]:
```suggestion
    n_spatial_filters : int, optional
```

File: braindecode/models/sccnet.py [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+        Number of spatial filters in the first convolutional layer. Default is 22.
+    n_filters_spat_filt : int, optional
```
Comment (sylvchev) [inline braindecode/models/sccnet.py:47]:
```suggestion
    n_spatial_filters2 : int, optional
```
or `n_spatial_filters_smooth`

File: braindecode/models/sccnet.py [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+        Dropout probability. Default is 0.5.
+    activation : nn.Module, optional
+        Activation function after the second convolutional layer. Default is
+        logarithm activation.
+
+    Notes
+    -----
+    This implementation is not guaranteed to be correct, has not been checked
+    by original authors, only reimplemented from the paper description and
+    source that we are not sure if it is correct [sccnetcode]_.
```
Comment (sylvchev) [inline braindecode/models/sccnet.py:60]:
```suggestion
    the source that have not been tested [sccnetcode]_.
```

**Review by sylvchev — APPROVED** [review]

**Comment by bruAristimunha** [comment]
Thank you for the review @sylvchev

**Pull request merged**